Detectors/GlobalTracking/include/GlobalTracking/TrackCuts.h

```diff
@@ -0,0 +1,128 @@
+  bool isSelected(GID trackIndex, o2::globaltracking::RecoContainer& data)
+  {
+    o2::track::TrackParCov trk;
+    auto contributorsGID = data.getSingleDetectorRefs(trackIndex);
+    auto src = trackIndex.getSource(); // make selections depending on track source
+    // ITS tracks
+    if (contributorsGID[GIndex::Source::ITS].isIndexSet()) { // ITS tracks selection
+      isBarrelTrack = true;
+      const auto& itsTrk = data.getITSTrack(contributorsGID[GID::ITS]);
+      int ITSnClusters = itsTrk.getNClusters();
+      float ITSchi2 = itsTrk.getChi2();
+      float itsChi2NCl = ITSnClusters != 0 ? ITSchi2 / (float)ITSnClusters : 0;
+      uint8_t itsClusterMap = itsTrk.getPattern();
+      if (ITSnClusters <= mMinNClustersITS ||
+          itsChi2NCl >= mMaxChi2PerClusterITS ||
+          TrackMethods::FulfillsITSHitRequirements(itsClusterMap, mRequiredITSHits) == false) {
+        return false;
+      }
+    }
+    // TPC tracks
+    if (contributorsGID[GIndex::Source::TPC].isIndexSet()) {
+      isBarrelTrack = true;
+      const auto& tpcTrk = data.getTPCTrack(contributorsGID[GID::TPC]);
+      // uint8_t tpcNClsShared, tpcNClsFound, tpcNClsCrossed, tpcNClsFindable, tpcChi2NCl;
+      // tpcNClsFindable = tpcTrk.getNClusters();
+      // tpcChi2NCl = tpcTrk.getNClusters() ? tpcTrk.getChi2() / tpcTrk.getNClusters() : 0;
+      // o2::TrackMethods::countTPCClusters(tpcTrk, data.getTPCTracksClusterRefs(), data.clusterShMapTPC, data.getTPCClusters(), tpcNClsShared, tpcNClsFound, tpcNClsCrossed);
+      // double tpcCrossedRowsOverFindableCls = tpcNClsCrossed / tpcNClsFindable;
+      math_utils::Point3D<float> v{};
+      std::array<float, 2> dca;
+      if (tpcTrk.getPt() < mPtTPCCut ||
+          std::abs(tpcTrk.getEta()) > mEtaTPCCut ||
+          tpcTrk.getNClusters() < mNTPCClustersCut ||
+          (!(const_cast<o2::tpc::TrackTPC&>(tpcTrk).propagateParamToDCA(v, mBz, &dca, mDCACut)) ||
+           std::abs(dca[0]) > mDCACutY)) {
+        return false;
+      }
+    }
+    // if (contributorsGID[GIndex::Source::TRD].isIndexSet()) {
+    //   const auto& trdTrk = data.getTrack<o2::trd::TrackTRD>(contributorsGID[GID::TRD]);
+    // }
+    // if (contributorsGID[GIndex::Source::TOF].isIndexSet()) {
+    //   const auto& tofMatch = data.getTOFMatch(trackIndex);
+    // }
+    // if (contributorsGID[GIndex::Source::TPCTRDTOF].isIndexSet()) {
+    //   trk = data.getTrack<o2::track::TrackParCov>(data.getTPCTRDTOFMatches()[trackIndex].getTrackRef());
+    // }
+    // if (contributorsGID[GIndex::Source::ITSTPCTRDTOF].isIndexSet()) {
+    //   trk = data.getTrack<o2::track::TrackParCov>(data.getITSTPCTRDTOFMatches()[trackIndex].getTrackRef()); // ITSTPCTRDTOF is ITSTPCTRD + TOF cluster
+    // }
+    // if (contributorsGID[GIndex::Source::ITSTPCTOF].isIndexSet()) {
+    //   trk = data.getTrack<o2::track::TrackParCov>(data.getTOFMatch(trackIndex).getTrackRef()); // ITSTPCTOF is ITSTPC + TOF cluster
+    // }
```

Comment:
if this code will not be used in future, please remove it.

Reply:
@shahor02  done!